electron/launcher.ts

```diff
@@ -240,18 +244,11 @@ function setupEnvVars(gameSettings: GameSettings) {
   if (gameSettings.audioFix) {
     ret.PULSE_LATENCY_MSEC = '60'
   }
-  if (gameSettings.otherOptions) {
```

Comment:
then we could use this only in the `else` for `if (gameSettings.envitonmentOptions)` to use it only as a fallback